galasa-parent/dev.galasa.framework.api.openapi/src/main/resources/openapi.yaml

```diff
@@ -12,15 +12,9 @@ info:
     url: https://galasa.dev/support
 
 servers:
-  - url: '/'
-  - url: 'http://{server}:{port}'
-    variables:
-      server:
-        default: localhost
-      port:
-        default: "8080"
+  - url: '{API_SERVER_URL}'
```

Comment:
I think it's worth commenting here to say where this value gets filled-in ?

Reply:
Done